includes/admin/bulk-actions.php

```diff
@@ -224,4 +225,34 @@ public function custom_bulk_admin_notices() {
 		}
 	}
 
+	/**
+	 * Update pro addon if install and version is less then 1.0.7.
+	 *
+	 * @since  3.1.37
+	 *
+	 */
+	public function simcal_update_pro_addon() {
+
+		if ( $screen = simcal_is_admin_screen() ) {
+
+			$plugins = get_plugins();
+
+			if (isset($plugins['Simple-Calendar-Google-Calendar-Pro/simple-calendar-google-calendar-pro.php'])) {
+
+				if ($plugins['Simple-Calendar-Google-Calendar-Pro/simple-calendar-google-calendar-pro.php']['Version'] <= '1.0.6' 
```

Comment:
This won't work since 1.0.6 is not decimal. You can use https://www.php.net/manual/en/function.version-compare.php.

Reply:
Added suggested PHP function.